pkg/kubelet/cm/helpers_linux.go

```diff
@@ -44,3 +154,36 @@ func GetCgroupSubsystems() (*CgroupSubsystems, error) {
 		MountPoints: mountPoints,
 	}, nil
 }
+
+// getCgroupProcs takes a cgroup directory name as an argument
+// reads through the cgroup's procs file and returns a list of tgid's.
+// It returns an empty list if a procs file doesn't exists
+func getCgroupProcs(dir string) ([]int, error) {
+	procsFile := filepath.Join(dir, "cgroup.procs")
+	_, err := os.Stat(procsFile)
+	if os.IsNotExist(err) {
+		// The procsFile does not exist, So no pids attached to this directory
+		return []int{}, nil
+	}
+	f, err := os.Open(procsFile)
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+
+	var (
+		s   = bufio.NewScanner(f)
+		out = []int{}
+	)
+
+	for s.Scan() {
+		if t := s.Text(); t != "" {
+			pid, err := strconv.Atoi(t)
+			if err != nil {
+				return nil, err
```

Comment:
The error from `Atoi` doesn't have enough context. Suggest `Unexpected line in $path; could not convert to pid: $err`

Not that this is going to happen, but still.